plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -237,14 +237,37 @@ export class HookCommander {
         this.statsd?.increment('webhook_firings')
     }
 
-    private async postRestHook(
+    public async postRestHook(
         hook: Hook,
         event: PluginEvent,
         person: CachedPersonData | Person | undefined
     ): Promise<void> {
+        let sendablePerson: Record<string, any> = {}
+        if (person) {
+            const {      
+                uuid,
+                properties,
+                team_id,
+                id
+            } = person
+
+            // CachedPersonData has created_at_iso (string), whereas Person has created_at (DateTime)
+            // so we standardize into ISO before sending the payload
```

Comment:
hmm... would it be safer/more futureproof if we made CachedPersonData have created_at (DateTime) instead?

Reply:
I thought that could be good - but I thought you had a good reason for storing it as ISO